{"timestamp":"2025-10-29T11:37:36.116389","level":"info","event":"DAG bundles loaded: dags-folder","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-10-29T11:37:36.118312","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/Airflow.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-10-29T11:37:39.645506","level":"info","event":"Tmp dir root location: /tmp","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:37:39.647306","level":"info","event":"Running command: ['/usr/bin/bash', '-c', 'python3 /opt/airflow/scripts/ARPEGE.py']","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:37:39.665975","level":"info","event":"Output:","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:37:45.596200","level":"info","event":"Unable to find image 'ghcr.io/open-meteo/open-meteo:latest' locally","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:37:47.607138","level":"info","event":"latest: Pulling from open-meteo/open-meteo","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:37:49.778896","level":"info","event":"38c489eb415c: Pulling fs layer","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:37:49.779186","level":"info","event":"1d26ee9223c9: Pulling fs layer","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:37:49.779385","level":"info","event":"590a1749d2aa: Pulling fs layer","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:37:49.779524","level":"info","event":"4f4fb700ef54: Pulling fs layer","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:37:49.779885","level":"info","event":"9423019a602f: Pulling fs layer","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:37:49.780079","level":"info","event":"5de5e513e911: Pulling fs layer","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:37:49.780221","level":"info","event":"4fb979fcde0c: Pulling fs layer","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:37:49.780354","level":"info","event":"891913cee06e: Pulling fs layer","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:37:49.780468","level":"info","event":"f7f4d240354a: Pulling fs layer","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:37:49.780576","level":"info","event":"a23bf04b88b3: Pulling fs layer","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:37:49.780683","level":"info","event":"215ed5a63843: Pulling fs layer","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:37:49.866074","level":"info","event":"4f4fb700ef54: Already exists","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:37:56.266014","level":"info","event":"38c489eb415c: Download complete","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:37:56.266540","level":"info","event":"891913cee06e: Download complete","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:37:56.268328","level":"info","event":"9423019a602f: Download complete","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:37:56.269128","level":"info","event":"5de5e513e911: Download complete","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:37:57.365811","level":"info","event":"590a1749d2aa: Download complete","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:38:06.764198","level":"info","event":"a23bf04b88b3: Download complete","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:38:09.863682","level":"info","event":"215ed5a63843: Download complete","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:38:14.362213","level":"info","event":"4fb979fcde0c: Download complete","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:38:17.086209","level":"info","event":"215ed5a63843: Pull complete","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:38:18.363157","level":"info","event":"1d26ee9223c9: Download complete","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:38:19.962640","level":"info","event":"f7f4d240354a: Download complete","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:38:21.284241","level":"info","event":"a23bf04b88b3: Pull complete","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:38:23.281644","level":"info","event":"1d26ee9223c9: Pull complete","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:38:37.166218","level":"info","event":"891913cee06e: Pull complete","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:38:37.179413","level":"info","event":"f7f4d240354a: Pull complete","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:38:37.263352","level":"info","event":"4f4fb700ef54: Pull complete","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:38:39.074681","level":"info","event":"4fb979fcde0c: Pull complete","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:38:39.166603","level":"info","event":"9423019a602f: Pull complete","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:38:39.270725","level":"info","event":"590a1749d2aa: Pull complete","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:38:39.364259","level":"info","event":"38c489eb415c: Pull complete","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:38:39.366441","level":"info","event":"5de5e513e911: Pull complete","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:38:39.374536","level":"info","event":"Digest: sha256:8d8f367e13caf3867536dec7b036b062715467a7215df0426c46f9d46362fb9d","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:38:39.375071","level":"info","event":"Status: Downloaded newer image for ghcr.io/open-meteo/open-meteo:latest","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:38:41.874489","level":"info","event":"[ INFO ] Checking for files to with more than 7 past days data","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:38:43.246550","level":"info","event":"[ INFO ] Downloading 4 files (60.8 MB)","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:38:43.246918","level":"info","event":"[ INFO ] Downloading file https://openmeteo.s3.amazonaws.com/data/meteofrance_arpege_europe/static/HSURF.om","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:38:43.247181","level":"info","event":"[ INFO ] Downloading file https://openmeteo.s3.amazonaws.com/data/meteofrance_arpege_europe/static/meta.json","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:38:43.474451","level":"info","event":"[ INFO ] Downloading file https://openmeteo.s3.amazonaws.com/data/meteofrance_arpege_europe/temperature_2m/chunk_2630.om","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:38:44.120830","level":"info","event":"[ INFO ] Downloading file https://openmeteo.s3.amazonaws.com/data/meteofrance_arpege_europe/temperature_2m/chunk_2631.om","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:39:03.808110","level":"info","event":"[ INFO ] Transfer 18% 6.1 MB / 34.2 MB in 20s, 312.8 KB/s remaining 92s","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:39:04.376362","level":"info","event":"[ INFO ] Transfer 23% 6.0 MB / 26.3 MB in 20s, 307.7 KB/s remaining 68s","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:39:23.867752","level":"info","event":"[ INFO ] Transfer 35% 12.0 MB / 34.2 MB in 40s, 300.9 KB/s remaining 76s","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:39:24.390568","level":"info","event":"[ INFO ] Transfer 45% 11.8 MB / 26.3 MB in 40s, 296.6 KB/s remaining 50s","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:39:43.868944","level":"info","event":"[ INFO ] Transfer 52% 18.0 MB / 34.2 MB in 60s, 306.0 KB/s remaining 54s","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:39:44.443353","level":"info","event":"[ INFO ] Transfer 67% 17.6 MB / 26.3 MB in 60s, 299.3 KB/s remaining 30s","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:40:03.952214","level":"info","event":"[ INFO ] Transfer 70% 23.9 MB / 34.2 MB in 80s, 306.0 KB/s remaining 35s","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:40:04.604779","level":"info","event":"[ INFO ] Transfer 90% 23.6 MB / 26.3 MB in 80s, 304.3 KB/s remaining 9.4s","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:40:13.747805","level":"info","event":"[ INFO ] meteofrance_arpege_europe 4 files 44% 26.6 MB / 60.8 MB in 91s, 302.2 KB/s remaining 117s","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:40:24.134599","level":"info","event":"[ INFO ] Transfer 88% 30.0 MB / 34.2 MB in 100s, 311.9 KB/s remaining 13.9s","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:40:38.046410","level":"info","event":"[ INFO ] meteofrance_arpege_europe 4 files 100% 60.8 MB / 60.8 MB in 115s, 1.43 MB/s remaining 0.0ms","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:40:38.046926","level":"info","event":"[ INFO ] meteofrance_arpege_europe 4 files completed 60.8 MB in 115s. Average speed 542.1 KB/s","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:40:38.047331","level":"info","event":"[ INFO ] Finished downloading 60.8 MB in 116s","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:40:40.182015","level":"info","event":"[ INFO ] Checking for files to with more than 7 past days data","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:40:41.582569","level":"info","event":"[ INFO ] Downloading 2 files (22.0 MB)","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:40:41.583966","level":"info","event":"[ INFO ] Downloading file https://openmeteo.s3.amazonaws.com/data/meteofrance_arpege_europe/precipitation/chunk_2630.om","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:40:41.584315","level":"info","event":"[ INFO ] Downloading file https://openmeteo.s3.amazonaws.com/data/meteofrance_arpege_europe/precipitation/chunk_2631.om","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:41:01.942795","level":"info","event":"[ INFO ] Transfer 64% 6.0 MB / 9.4 MB in 20s, 307.7 KB/s remaining 11.2s","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:41:02.012692","level":"info","event":"[ INFO ] Transfer 47% 6.0 MB / 12.7 MB in 20s, 306.0 KB/s remaining 23s","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:41:13.363520","level":"info","event":"[ INFO ] meteofrance_arpege_europe 2 files 42% 9.4 MB / 22.0 MB in 32s, 309.2 KB/s remaining 43s","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:41:22.152400","level":"info","event":"[ INFO ] Transfer 94% 11.9 MB / 12.7 MB in 40s, 304.3 KB/s remaining 2.57s","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:41:24.576387","level":"info","event":"[ INFO ] meteofrance_arpege_europe 2 files completed 22.0 MB in 43s. Average speed 524.9 KB/s","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:41:24.576813","level":"info","event":"[ INFO ] Finished downloading 22.1 MB in 44s","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:41:25.664542","level":"info","event":"[ INFO ] Checking for files to with more than 7 past days data","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:41:27.011216","level":"info","event":"[ INFO ] Downloading 2 files (46.0 MB)","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:41:27.012292","level":"info","event":"[ INFO ] Downloading file https://openmeteo.s3.amazonaws.com/data/meteofrance_arpege_europe/relative_humidity_2m/chunk_2631.om","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:41:27.012550","level":"info","event":"[ INFO ] Downloading file https://openmeteo.s3.amazonaws.com/data/meteofrance_arpege_europe/relative_humidity_2m/chunk_2630.om","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:41:47.370065","level":"info","event":"[ INFO ] Transfer 23% 6.0 MB / 26.1 MB in 20s, 308.5 KB/s remaining 67s","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:41:47.374802","level":"info","event":"[ INFO ] Transfer 30% 5.9 MB / 19.9 MB in 20s, 303.4 KB/s remaining 47s","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:42:07.401670","level":"info","event":"[ INFO ] Transfer 46% 12.1 MB / 26.1 MB in 40s, 311.1 KB/s remaining 46s","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:42:07.423403","level":"info","event":"[ INFO ] Transfer 60% 11.9 MB / 19.9 MB in 40s, 303.5 KB/s remaining 27s","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:42:27.453269","level":"info","event":"[ INFO ] Transfer 69% 18.0 MB / 26.1 MB in 60s, 302.6 KB/s remaining 27s","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:42:27.513673","level":"info","event":"[ INFO ] Transfer 89% 17.7 MB / 19.9 MB in 60s, 301.7 KB/s remaining 7.2s","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:42:35.258523","level":"info","event":"[ INFO ] meteofrance_arpege_europe 2 files 43% 19.9 MB / 46.0 MB in 68s, 299.1 KB/s remaining 90s","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:42:47.609685","level":"info","event":"[ INFO ] Transfer 92% 23.9 MB / 26.1 MB in 80s, 302.6 KB/s remaining 7.5s","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:42:54.862931","level":"info","event":"[ INFO ] meteofrance_arpege_europe 2 files completed 46.0 MB in 88s. Average speed 535.8 KB/s","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:42:54.863338","level":"info","event":"[ INFO ] Finished downloading 46.0 MB in 89s","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:42:56.023143","level":"info","event":"[ INFO ] Checking for files to with more than 7 past days data","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:42:57.388594","level":"info","event":"[ INFO ] Downloading 2 files (59.6 MB)","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:42:57.390010","level":"info","event":"[ INFO ] Downloading file https://openmeteo.s3.amazonaws.com/data/meteofrance_arpege_europe/shortwave_radiation/chunk_2631.om","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:42:57.390417","level":"info","event":"[ INFO ] Downloading file https://openmeteo.s3.amazonaws.com/data/meteofrance_arpege_europe/shortwave_radiation/chunk_2630.om","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:43:17.777050","level":"info","event":"[ INFO ] Transfer 17% 5.9 MB / 34.5 MB in 20s, 301.0 KB/s remaining 98s","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:43:17.786309","level":"info","event":"[ INFO ] Transfer 23% 5.9 MB / 25.1 MB in 20s, 300.9 KB/s remaining 66s","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:43:37.794787","level":"info","event":"[ INFO ] Transfer 47% 11.9 MB / 25.1 MB in 40s, 307.7 KB/s remaining 44s","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:43:37.865018","level":"info","event":"[ INFO ] Transfer 35% 12.0 MB / 34.5 MB in 40s, 311.1 KB/s remaining 75s","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:43:57.846992","level":"info","event":"[ INFO ] Transfer 72% 18.0 MB / 25.1 MB in 60s, 311.1 KB/s remaining 24s","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:43:57.983341","level":"info","event":"[ INFO ] Transfer 52% 18.0 MB / 34.5 MB in 60s, 311.0 KB/s remaining 55s","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:17.921471","level":"info","event":"[ INFO ] Transfer 95% 23.9 MB / 25.1 MB in 80s, 304.3 KB/s remaining 4.03s","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:18.009730","level":"info","event":"[ INFO ] Transfer 70% 24.0 MB / 34.5 MB in 80s, 306.8 KB/s remaining 35s","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:21.787030","level":"info","event":"[ INFO ] meteofrance_arpege_europe 2 files 42% 25.1 MB / 59.6 MB in 84s, 306.0 KB/s remaining 116s","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:38.158984","level":"info","event":"[ INFO ] Transfer 87% 30.2 MB / 34.5 MB in 100s, 315.4 KB/s remaining 14.2s","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.247898","level":"info","event":"[ INFO ] meteofrance_arpege_europe 2 files 100% 59.6 MB / 59.6 MB in 115s, 1.15 MB/s remaining 0.0ms","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.248245","level":"info","event":"[ INFO ] meteofrance_arpege_europe 2 files completed 59.6 MB in 115s. Average speed 531.5 KB/s","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.248513","level":"info","event":"[ INFO ] Finished downloading 59.7 MB in 116s","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.833749","level":"info","event":"🔄 Descargando datos de http://127.0.0.1:8081/v1/forecast ...","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.834274","level":"info","event":"⬇️ Descargando temperature_2m del Bucket S3 de AWS...","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.834653","level":"info","event":"⬇️ Descargando precipitation del Bucket S3 de AWS...","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.835095","level":"info","event":"⬇️ Descargando relative_humidity_2m del Bucket S3 de AWS...","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.835445","level":"info","event":"⬇️ Descargando shortwave_radiation del Bucket S3 de AWS...","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.835815","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=39.1281556&longitude=-1.5212770999999998&models=meteofrance_arpege_europe&hourly=temperature_2m (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x707802853b30>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.836407","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=39.1281556&longitude=-1.5212770999999998&models=meteofrance_arpege_europe&hourly=precipitation (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7078019854f0>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.836738","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=39.1281556&longitude=-1.5212770999999998&models=meteofrance_arpege_europe&hourly=relative_humidity_2m (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7078019858b0>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.837142","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=39.1281556&longitude=-1.5212770999999998&models=meteofrance_arpege_europe&hourly=shortwave_radiation (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7078019871a0>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.837482","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=39.1281556&longitude=-0.5212770999999998&models=meteofrance_arpege_europe&hourly=temperature_2m (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x707801987950>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.837733","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=39.1281556&longitude=-0.5212770999999998&models=meteofrance_arpege_europe&hourly=precipitation (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x70780178c1d0>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.838001","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=39.1281556&longitude=-0.5212770999999998&models=meteofrance_arpege_europe&hourly=relative_humidity_2m (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x707801987980>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.838521","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=39.1281556&longitude=-0.5212770999999998&models=meteofrance_arpege_europe&hourly=shortwave_radiation (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7078019870b0>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.838854","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=39.1281556&longitude=-1.5212770999999998&models=meteofrance_arpege_europe&hourly=temperature_2m (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7078019857c0>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.840014","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=39.1281556&longitude=-1.5212770999999998&models=meteofrance_arpege_europe&hourly=precipitation (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x707802d010d0>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.840502","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=39.1281556&longitude=-1.5212770999999998&models=meteofrance_arpege_europe&hourly=relative_humidity_2m (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x70780178c4d0>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.840788","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=39.1281556&longitude=-1.5212770999999998&models=meteofrance_arpege_europe&hourly=shortwave_radiation (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x70780178cf20>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.841098","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=39.1281556&longitude=-0.5212770999999998&models=meteofrance_arpege_europe&hourly=temperature_2m (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x707803468b60>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.841679","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=39.1281556&longitude=-0.5212770999999998&models=meteofrance_arpege_europe&hourly=precipitation (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x707801985550>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.842165","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=39.1281556&longitude=-0.5212770999999998&models=meteofrance_arpege_europe&hourly=relative_humidity_2m (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7078019869c0>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.842432","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=39.1281556&longitude=-0.5212770999999998&models=meteofrance_arpege_europe&hourly=shortwave_radiation (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x707801987a10>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.842741","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=39.1281556&longitude=-1.5212770999999998&models=meteofrance_arpege_europe&hourly=temperature_2m (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x70780178cad0>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.843195","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=39.1281556&longitude=-1.5212770999999998&models=meteofrance_arpege_europe&hourly=precipitation (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x70780178c800>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.843468","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=39.1281556&longitude=-1.5212770999999998&models=meteofrance_arpege_europe&hourly=relative_humidity_2m (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x707804d209b0>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.843819","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=39.1281556&longitude=-1.5212770999999998&models=meteofrance_arpege_europe&hourly=shortwave_radiation (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x707801987980>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.844152","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=39.1281556&longitude=-0.5212770999999998&models=meteofrance_arpege_europe&hourly=temperature_2m (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x707801987200>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.906736","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=39.1281556&longitude=-0.5212770999999998&models=meteofrance_arpege_europe&hourly=precipitation (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x707801985610>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.907123","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=39.1281556&longitude=-0.5212770999999998&models=meteofrance_arpege_europe&hourly=relative_humidity_2m (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x70780178d790>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.907300","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=39.1281556&longitude=-0.5212770999999998&models=meteofrance_arpege_europe&hourly=shortwave_radiation (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x70780178ce30>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.907488","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=39.1281556&longitude=-1.5212770999999998&models=meteofrance_arpege_europe&hourly=temperature_2m (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x70780178db80>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.907626","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=39.1281556&longitude=-1.5212770999999998&models=meteofrance_arpege_europe&hourly=precipitation (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x707801985430>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.907794","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=39.1281556&longitude=-1.5212770999999998&models=meteofrance_arpege_europe&hourly=relative_humidity_2m (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7078019872c0>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.907936","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=39.1281556&longitude=-1.5212770999999998&models=meteofrance_arpege_europe&hourly=shortwave_radiation (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x707801987950>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.908051","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=39.1281556&longitude=-0.5212770999999998&models=meteofrance_arpege_europe&hourly=temperature_2m (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x70783366ac90>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.908244","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=39.1281556&longitude=-0.5212770999999998&models=meteofrance_arpege_europe&hourly=precipitation (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x70780178c140>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.908567","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=39.1281556&longitude=-0.5212770999999998&models=meteofrance_arpege_europe&hourly=relative_humidity_2m (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x70780178d9d0>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.908863","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=39.1281556&longitude=-0.5212770999999998&models=meteofrance_arpege_europe&hourly=shortwave_radiation (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x707802d16090>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.909234","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=40.1281556&longitude=-1.5212770999999998&models=meteofrance_arpege_europe&hourly=temperature_2m (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x707801987b90>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.909524","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=40.1281556&longitude=-1.5212770999999998&models=meteofrance_arpege_europe&hourly=precipitation (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x707801987d70>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.909749","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=40.1281556&longitude=-1.5212770999999998&models=meteofrance_arpege_europe&hourly=relative_humidity_2m (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7078019873e0>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.910007","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=40.1281556&longitude=-1.5212770999999998&models=meteofrance_arpege_europe&hourly=shortwave_radiation (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x70780178d3a0>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.910228","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=40.1281556&longitude=-0.5212770999999998&models=meteofrance_arpege_europe&hourly=temperature_2m (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x70780178dd30>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.910438","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=40.1281556&longitude=-0.5212770999999998&models=meteofrance_arpege_europe&hourly=precipitation (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x707801986c00>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.910690","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=40.1281556&longitude=-0.5212770999999998&models=meteofrance_arpege_europe&hourly=relative_humidity_2m (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x707801987770>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.911007","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=40.1281556&longitude=-0.5212770999999998&models=meteofrance_arpege_europe&hourly=shortwave_radiation (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x707804c7a420>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.911282","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=40.1281556&longitude=-1.5212770999999998&models=meteofrance_arpege_europe&hourly=temperature_2m (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x70780178c8f0>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.911523","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=40.1281556&longitude=-1.5212770999999998&models=meteofrance_arpege_europe&hourly=precipitation (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x70780178c3b0>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.911754","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=40.1281556&longitude=-1.5212770999999998&models=meteofrance_arpege_europe&hourly=relative_humidity_2m (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x70780178e7e0>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.911987","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=40.1281556&longitude=-1.5212770999999998&models=meteofrance_arpege_europe&hourly=shortwave_radiation (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x707801af8860>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.959903","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=40.1281556&longitude=-0.5212770999999998&models=meteofrance_arpege_europe&hourly=temperature_2m (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x707801987dd0>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.960342","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=40.1281556&longitude=-0.5212770999999998&models=meteofrance_arpege_europe&hourly=precipitation (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x707801985820>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.960591","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=40.1281556&longitude=-0.5212770999999998&models=meteofrance_arpege_europe&hourly=relative_humidity_2m (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x707802c32120>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.960906","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=40.1281556&longitude=-0.5212770999999998&models=meteofrance_arpege_europe&hourly=shortwave_radiation (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x70780178d010>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.961142","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=40.1281556&longitude=-1.5212770999999998&models=meteofrance_arpege_europe&hourly=temperature_2m (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x70780178f230>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.961349","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=40.1281556&longitude=-1.5212770999999998&models=meteofrance_arpege_europe&hourly=precipitation (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7078019866f0>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.961600","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=40.1281556&longitude=-1.5212770999999998&models=meteofrance_arpege_europe&hourly=relative_humidity_2m (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x707801986f90>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.961809","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=40.1281556&longitude=-1.5212770999999998&models=meteofrance_arpege_europe&hourly=shortwave_radiation (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x707801987c20>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.962111","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=40.1281556&longitude=-0.5212770999999998&models=meteofrance_arpege_europe&hourly=temperature_2m (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x707801af6cc0>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.962303","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=40.1281556&longitude=-0.5212770999999998&models=meteofrance_arpege_europe&hourly=precipitation (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x70780178edb0>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.962586","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=40.1281556&longitude=-0.5212770999999998&models=meteofrance_arpege_europe&hourly=relative_humidity_2m (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x70780178c740>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.962839","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=40.1281556&longitude=-0.5212770999999998&models=meteofrance_arpege_europe&hourly=shortwave_radiation (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x707802862f30>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.963150","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=40.1281556&longitude=-1.5212770999999998&models=meteofrance_arpege_europe&hourly=temperature_2m (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x707801986a20>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.963390","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=40.1281556&longitude=-1.5212770999999998&models=meteofrance_arpege_europe&hourly=precipitation (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x707801987b30>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.963770","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=40.1281556&longitude=-1.5212770999999998&models=meteofrance_arpege_europe&hourly=relative_humidity_2m (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7078019866f0>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.964148","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=40.1281556&longitude=-1.5212770999999998&models=meteofrance_arpege_europe&hourly=shortwave_radiation (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x70780178c9b0>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.964650","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=40.1281556&longitude=-0.5212770999999998&models=meteofrance_arpege_europe&hourly=temperature_2m (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x70780178f020>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.964976","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=40.1281556&longitude=-0.5212770999999998&models=meteofrance_arpege_europe&hourly=precipitation (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x707802d16090>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.965163","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=40.1281556&longitude=-0.5212770999999998&models=meteofrance_arpege_europe&hourly=relative_humidity_2m (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x707801986930>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.965297","level":"info","event":"HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /v1/forecast?latitude=40.1281556&longitude=-0.5212770999999998&models=meteofrance_arpege_europe&hourly=shortwave_radiation (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7078019871a0>: Failed to establish a new connection: [Errno 111] Connection refused'))","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.965422","level":"info","event":"","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.965569","level":"info","event":"🏁 Descarga completada en 427.60 s (7.13 min)","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.965708","level":"info","event":"❌ No se obtuvieron datos.","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.965931","level":"info","event":"Traceback (most recent call last):","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.966139","level":"info","event":"  File \"/opt/airflow/scripts/ARPEGE.py\", line 177, in <module>","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.966423","level":"info","event":"    main()","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.966629","level":"info","event":"  File \"/opt/airflow/scripts/ARPEGE.py\", line 157, in main","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.966778","level":"info","event":"    df = df.dropna(subset=VARIABLES) # Borra las filas con datos de las variables faltantes que corresponden a predicciones no disponibles.","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.966970","level":"info","event":"         ^^^^^^^^^^^^^^^^^^^^^^^^^^^","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.967136","level":"info","event":"  File \"/home/airflow/.local/lib/python3.12/site-packages/pandas/core/frame.py\", line 6418, in dropna","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.967350","level":"info","event":"    raise KeyError(np.array(subset)[check].tolist())","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:52.967552","level":"info","event":"KeyError: ['temperature_2m', 'precipitation', 'relative_humidity_2m', 'shortwave_radiation']","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:53.221543","level":"info","event":"Command exited with return code 1","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T11:44:53.223028","level":"error","event":"Task failed with exception","logger":"task","error_detail":[{"exc_type":"AirflowException","exc_value":"Bash command failed. The command returned a non-zero exit code 1.","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":920,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1215,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":397,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/bash.py","lineno":232,"name":"execute"}],"is_group":false,"exceptions":[]}]}
